mozilla/blurts-server
# Add name_suffix and middle_name to onerep_profiles table (MNTOR-3144)

# References: 
Jira: [MNTOR-3144](https://mozilla-hub.atlassian.net/browse/MNTOR-3144)



# Description

Adds `name_suffix` and  `middle_name` columns to the `onerep_profiles` table that is needed for additional information a user can optionally provide for their data broker scan that will be introduced in [MNTOR-2813](https://mozilla-hub.atlassian.net/browse/MNTOR-2813). The additions to the `/scan` endpoint to include these new values will be introduced together with the updates to the UI.

# How to test

1. Run the [DB migration](https://github.com/mozilla/blurts-server/compare/mntor-3144?expand=1#diff-96d29caca9c36ba64ce9f341d4442004a1520fc356a2ba189f1e4367ac15443a) introduced with this PR
2. The columns `name_suffix` and  `middle_name` should have been added to `onerep_profiles`. For existing users, the value will be `NULL`.

[MNTOR-3144]: https://mozilla-hub.atlassian.net/browse/MNTOR-3144?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[MNTOR-2813]: https://mozilla-hub.atlassian.net/browse/MNTOR-2813?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
